.github/workflows/fetch-data.yml

```
@@ -175,8 +188,46 @@ jobs:
+          # ブランチをプッシュ
+          git push origin "$BRANCH_NAME"
+
+          # Pull Request の作成
+          PR_TITLE="$COMMIT_MSG"
+
+          # gh CLIを使用してPRを作成（本文はコマンドラインで直接指定）
+          PR_URL=$(gh pr create \
+            --title "$PR_TITLE" \
+            --body "## 🤖 自動データ更新
+
+このPull Requestは GitHub Actions により自動生成されました。
```

actionlint (1.7.8)

could not parse as YAML: could not find expected ':'

(syntax-check)